Header header
string id
geometry_msgs/Point position
float32 influence_distance
